tests/main/views/test_direct_award.py

```diff
@@ -389,3 +389,45 @@ def test_search_api_urls_convert_to_correct_frontend_urls(self, search_api_url,
         doc = html.fromstring(body)
         assert html_escape(frontend_url) in body
         assert len(doc.xpath('//a[@href="{}"]'.format(frontend_url)))
+
+
+class TestDirectAwardEndSearch(TestDirectAwardBase):
+    def test_end_search_page_renders(self):
+        self.login_as_buyer()
+
+        res = self.client.get('/buyers/direct-award/g-cloud/projects/1/end-search')
+        assert res.status_code == 200
+
+        doc = html.fromstring(res.get_data(as_text=True))
+        assert len(doc.xpath('//h1[contains(normalize-space(), "End search")]')) == 1
+
+    def test_end_search_page_renders_error_when_results_more_than_999(self):
```

Comment:
The limit is 100 now, so we need to rename this test and update the return value later on. Also I think it's 100 inclusive (at least that's what your earlier logic validates).